.nvmrc

```diff
@@ -1 +1 @@
-v14
+v16
```

Comment:
We are [running CI tests on Node 16](https://github.com/hashicorp/react-components/pull/1022/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fL30), so we should also set nvm to at least 16 as well.